### PR TITLE
Edited the choices variable in runAqa script

### PR DESCRIPTION
**Pull request by 2001asjad**
Instead of hardcoding each subdir in the list, I have made a function that queries the contents of external and creates a list of acceptable build_list values. 

Fixes #2482

### CODE REVIEW

**Comment by 2001asjad** [comment]
> Instead of hardcoding each subdir in the list, I have made a function that queries the contents of external and creates a list of acceptable build_list values.
> 
> Fixes #2482

@smlambert Can you please take a look and drop out suggestions and corrections to this.
Thank You

**Comment by smlambert** [comment]
run aqa --help

**Comment by smlambert** [comment]
run aqa --sdk_resource nightly --build_list external/jacoco --target _jacoco_test --jdk_version 8 --jdk_impl hotspot --platform x86-64_linux

**Comment by smlambert** [comment]
I guess my attempt to run in the above comment will not use the python script from your development branch.

**Comment by 2001asjad** [comment]
> I guess my attempt to run in the above comment will not use the python script from your development branch.

Any suggestions @smlambert ??

**Comment by smlambert** [comment]
Can you check if you can run this script based on a comment on a PR to your own [branch](https://github.com/2001asjad/TKG/tree/asjad-issue-2482) ?  (as in create a 'dummy' PR to edit something against your branch and then try to type same comments as I did above in it as a comment)

**Comment by smlambert** [comment]
@renfeiw - can you advise, is that how you tested the auto exclude feature, in your branch?

**Comment by renfeiw** [comment]
You can test it as suggested in the PR: https://github.com/adoptium/TKG/pull/172#issue-609999348 
Note for testing: The runAqaArgParse.py script is expected to be in this repository on the master branch (adoptium/TKG:master), but does not yet exist because this PR has not been merged. Therefore, for testing, line 29 of runAqa.yml should be changed match your/my personal repo which has the runAqaArgParse.py script in the correct location (scripts/testBot)

**Comment by 2001asjad** [comment]
run aqa --help

**Comment by 2001asjad** [comment]
run aqa --sdk_resource nightly --build_list external/jacoco --target _jacoco_test --jdk_version 8 --jdk_impl hotspot --platform x86-64_linux

**Comment by 2001asjad** [comment]
run aqa --help

**Comment by 2001asjad** [comment]
run aqa --help

**Comment by 2001asjad** [comment]
Hello, @smlambert @renfeiw. The builds still did not start even I addressed the changes as mentioned above by @renfeiw .
github
 I have attached the ss where I addressed the changes.
Please look into it.
Thank You.

**Comment by 2001asjad** [comment]
run aqa --help

**Comment by smlambert** [comment]
@2001asjad - the suggestion was _not_ to try and run the script in this PR that is against master branch, but to create a dummy PR against your branch ([2001asjad:asjad-issue-2482](https://github.com/2001asjad/TKG/tree/asjad-issue-2482)) to be able to add comments to it, as it will run your edited python script and allow you to test the changes.

**Comment by 2001asjad** [comment]
> @2001asjad - the suggestion was _not_ to try and run the script in this PR that is against master branch, but to create a dummy PR against your branch ([2001asjad:asjad-issue-2482](https://github.com/2001asjad/TKG/tree/asjad-issue-2482)) to be able to add comments to it, as it will run your edited python script and allow you to test the changes.

Ok let me go through this and I'll let you know.. 
Thank You so much by the way

**Comment by renfeiw** [comment]
Could you please try the following: amend your commit modifying the line 29 of runAqa.yml to match your/my personal repo and trigger the tests. Once the tests are successful, you can amend your commit again to revert the change from yml file.

**Comment by 2001asjad** [comment]
run aqa --help

**Comment by 2001asjad** [comment]
> Could you please try the following: amend your commit modifying the line 29 of runAqa.yml to match your/my personal repo and trigger the tests. Once the tests are successful, you can amend your commit again to revert the change from yml file.

Did it above, but the builds aren't starting.

File: .github/workflows/runAqa.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
     - name: Checkout main TKG repo
       uses: actions/checkout@v2
       with:
-        repository: 'adoptium/TKG.git'
+        repository: '2001asjad/TKG.git'
```
Comment (karianna) [inline .github/workflows/runAqa.yml:29]: that should remain as adoptium I suspect.

**Review by karianna — CHANGES_REQUESTED** [review]

**Comment by 2001asjad** [comment]
@smlambert taking note of what @karianna suggested above, I am gonna try this, but I deleted my previous fork and now I think I have to do this again via new branch, gonna update you soon.

**Pull request closed without merging**